include/oneapi/dpl/internal/gcd_impl.h

```diff
@@ -0,0 +1,108 @@
+/*
+ *  Copyright (c) Intel Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#ifndef DPSTD_GCD
```

Comment:
It's better to have `_ONEDPL_...` macro here since we were agreed that all internal macros started with `_ONEDPL_...`. DPSTD is outdated name

Reply:
done.